src/test/database-remove.spec.js

```diff
@@ -0,0 +1,159 @@
+"use strict";
```

Comment:
The test file structure should mirror the source file structure, so this should be `src/test/database/remove.spec.js`

Reply:
Also, did you intend to commit this file? It looks like it's just a copy-paste of another test file and isn't actually testing any of the functionality you coded here...

Reply:
Oh, sorry, I did not meant to commit at that point.
I have just written tests. It should be ready for review now.